source/fundamentals/indexes.txt

```diff
@@ -186,14 +211,101 @@ Strings), and ``title`` fields:
    :language: kotlin
 
-Multikey indexes behave differently from non-multikey indexes in terms of query coverage, index bound computation, and
-sort behavior. For a full explanation of multikey indexes, including a discussion of their behavior and limitations,
-refer to the :manual:`Multikey Indexes page </core/index-multikey>` in the MongoDB manual.
+Multikey indexes behave differently from other indexes in terms of query coverage, index-bound computation, and
+sort behavior. To learn more about multikey indexes, including a discussion of their behavior and limitations,
+see :manual:`Multikey Indexes </core/index-multikey>` in the Server manual.
+
+.. _kotlin-search-indexes:
+
+Atlas Search Indexes
+~~~~~~~~~~~~~~~~~~~~
+
+The Atlas Search feature enables you to perform full-text searches on
+collections hosted on MongoDB Atlas. The indexes specify how you can
+perform full-text searches on specific fields.
+
+To learn more about MongoDB Atlas Search, see the
+:atlas:`Atlas Search Indexes </atlas-search/atlas-search-overview/#fts-indexes>`
+documentation.
+
+You can call the following methods on a collection to manage your Atlas Search
+indexes:
+
+- ``createSearchIndex()``
+- ``createSearchIndexes()``
+- ``listSearchIndexes()``
+- ``updateSearchIndex()``
+- ``dropSearchIndex()``
+
+.. note::
+
+   The Atlas Search index-management methods run asynchronously. The
+   driver methods can return before confirming that they ran
+   successfully. To determine the current status of the indexes, call the
+   ``listSearchIndexes()`` method.
+
+The following sections provide code examples that demonstrate how to use
+each of the preceding methods.
+
+Create a Search Index
++++++++++++++++++++++
+
+You can use the `createSearchIndex() <{+api+}/apidocs/mongodb-driver-kotlin-coroutine/mongodb-driver-kotlin-coroutine/com.mongodb.kotlin.client.coroutine/-mongo-collection/create-search-index.html>`__
+and `createSearchIndexes() <{+api+}/apidocs/mongodb-driver-kotlin-coroutine/mongodb-driver-kotlin-coroutine/com.mongodb.kotlin.client.coroutine/-mongo-collection/create-search-indexes.html>`__
+methods to create Atlas Search indexes on a collection.
+
+The following code example shows how to create a single index:
+
+.. literalinclude:: /examples/generated/SearchIndexesTest.snippet.single-search-index-create.kt
+   :language: kotlin
+
+The following code example shows how to create multiple indexes:
+
+.. literalinclude:: /examples/generated/SearchIndexesTest.snippet.multi-search-index-create.kt
+   :language: kotlin
+
+List Search Indexes
++++++++++++++++++++
+
+You can use the
+`listSearchIndexes() <{+api+}/apidocs/mongodb-driver-kotlin-coroutine/mongodb-driver-kotlin-coroutine/com.mongodb.kotlin.client.coroutine/-mongo-collection/list-search-indexes.html>`__
+method to return a list of the Atlas Search indexes on a collection.
+
+The following code example shows how to print a list of the search indexes of
+a collection:
```

Comment:
Based on [your comment](https://github.com/mongodb/docs-kotlin/pull/144#discussion_r1453607066), I thought you were changing from "of" to "on" (just making sure you didn't intend to make this change)

Reply:
AH, changed in one place but not the other.